test/external/CMakeLists.txt

```diff
@@ -2,5 +2,23 @@
 # Add tests based on external repositories
 #
 
+include(FetchContent)
+
+FetchContent_Declare(
+    ringtest
+    GIT_REPOSITORY https://github.com/neuronsimulator/ringtest
+    GIT_SHALLOW TRUE
+    SOURCE_DIR ${PROJECT_SOURCE_DIR}/external/tests/ringtest
+)
+
+FetchContent_Declare(
+    testcorenrn
+    GIT_REPOSITORY https://github.com/neuronsimulator/testcorenrn
+    GIT_SHALLOW TRUE
+    SOURCE_DIR ${PROJECT_SOURCE_DIR}/external/tests/testcorenrn
+)
```

Comment:
@alkino : I think we should use GIT_TAG as well? Because if I change the branch then most likely I want to fetch different/older/compatible version. Tagging @olupton for the review as he wrote #1346 .

Reply:
the right question is which tag?

Reply:
I would say the same commit (hash) that the submodule pointed to.